rental.rb

```diff
@@ -0,0 +1,14 @@
+class Rental
+  def initialize(date, book, person)
+    @date = date
+
+    @book = book
+    book.rentals << self
+
+    @person = person
+    person.rentals << self
+  end
+
+  attr_accessor :date
+  attr_reader :book, :person
```

Comment:
- [ ] Kindly add the getters and setters at the top of the `def initialize` method